src/engine/CMakeLists.txt

```diff
@@ -1,6 +1,5 @@
 add_library(engine
         Engine.h Engine.cpp
-        IndexSequence.h
```

Comment:
Your Commit Message says "We want to keep our own IndexSequence.h". How does this relate to erasing this line here?